### PR TITLE
EDU-2644: Expands graceful failover to mention retry period

### PR DESCRIPTION
**Pull request by fairlydurable**
Original source of truth @laniehei.
After she went on vacation, consulted with Nikita and team MRN.
New source of truth is Liang Mei.

### CODE REVIEW

**Comment by fairlydurable** [comment]
@laniehei Please review for correctness by end of business 14 June so I can pass this to team review. Thanks.

File: docs/production-deployment/cloud/multi-region.mdx [hunk 1]
```diff
@@ -144,7 +144,8 @@ This mode prioritizes _availability_ over consistency.
 #### Hybrid failover mode {#hybrid-failover}
 
 While graceful failovers are preferred for consistency, they aren’t always practical.
-Temporal Cloud’s hybrid failover mode (the default mode) limits an initial graceful failover attempt to 10 seconds or less.
+Temporal Cloud’s hybrid failover mode (the default mode) limits a single graceful failover attempt to 10 seconds or less.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/multi-region.mdx:147]:
```suggestion
Temporal Cloud’s hybrid failover mode (the default mode) limits an initial graceful failover attempt to 10 seconds or less.
```

File: docs/production-deployment/cloud/multi-region.mdx [hunk 2]
```diff
@@ -129,6 +129,7 @@ This prevents the loss of progress and avoids data conflicts.
 The Namespace experiences a short period of unavailability, defaulting to 10 seconds.
 
 During this period, existing Workflows stop progress.
+Temporal Cloud will return a "Service unavailable error", which is retried by SDKs.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/multi-region.mdx:132]:
```suggestion
Temporal Cloud will return a "Service unavailable error", which will be retried by SDKs.
```

File: docs/production-deployment/cloud/multi-region.mdx [hunk 3]
```diff
@@ -129,6 +129,7 @@ This prevents the loss of progress and avoids data conflicts.
 The Namespace experiences a short period of unavailability, defaulting to 10 seconds.
 
 During this period, existing Workflows stop progress.
+Temporal Cloud will return a "Service unavailable error", which will be retried by SDKs.
```
Comment (flossypurse) [inline docs/production-deployment/cloud/multi-region.mdx:132]:
```suggestion
Temporal Cloud returns a "Service unavailable error", which is retried by SDKs.
```

I **think** the style guidance says to use present tense in this situation.

**Review by flossypurse — APPROVED** [review]

File: docs/production-deployment/cloud/multi-region.mdx (hunk 3 above)
Comment (flossypurse) [reply docs/production-deployment/cloud/multi-region.mdx:132]: If you adopt here, suggest adopting for the other sentence below.

File: docs/production-deployment/cloud/multi-region.mdx [hunk 4]
```diff
@@ -145,6 +146,8 @@ This mode prioritizes _availability_ over consistency.
 
 While graceful failovers are preferred for consistency, they aren’t always practical.
 Temporal Cloud’s hybrid failover mode (the default mode) limits an initial graceful failover attempt to 10 seconds or less.
+During this period, existing Workflows stop progress.
+Temporal Cloud will return a "Service unavailable error", which will be retried by the SDKs.
```
Comment (fairlydurable) [inline docs/production-deployment/cloud/multi-region.mdx:150]:
```suggestion
Temporal Cloud returns a "Service unavailable error", which is retried by SDKs.
```

**Pull request merged**